src/Console/ConsoleOutput.php

```diff
@@ -147,13 +152,18 @@ class ConsoleOutput
         'alert' => ['text' => 'red'],
         'critical' => ['text' => 'red'],
         'error' => ['text' => 'red'],
+        'error.bg' => ['background' => 'red', 'text' => 'black'],
         'warning' => ['text' => 'yellow'],
+        'warning.bg' => ['background' => 'yellow', 'text' => 'black'],
         'info' => ['text' => 'cyan'],
+        'info.bg' => ['background' => 'cyan', 'text' => 'black'],
```

Comment:
A bright color for info would look like a result to me. Could we try grey with cyan text?

Reply:
We don't have grey right now, as we only do the basic 8 color ansi palette. We could use `white` as the background, or not include a banner style for info :shrug:

Reply:
White would work.